h/logging.h

```diff
@@ -297,15 +317,23 @@ bool logShouldTraceInternal(LoggingContext *context, uint64 componentID, int lev
 
 /* this log message will be sent to the destination associated to the component 
  */
+void _zowelog(LoggingContext *context, uint64 compID, char* path, int line, int level, char *formatString, ...);
+void _zowedump(LoggingContext *context, uint64 compID, int level, void *data, int dataSize, char* path, int line);
 
-void zowelog(LoggingContext *context, uint64 compID, int level, char *formatString, ...);
-void zowedump(LoggingContext *context, uint64 compID, int level, void *data, int dataSize);
-
+char *standardDumperFunction(char *workBuffer, int workBufferSize,
+                                    void *data, int dataSize, int lineNumber);
 #define LOGCHECK(context,component,level) \
   ((component > MAX_LOGGING_COMPONENTS) ? \
    (context->applicationComponents[component].level >= level) : \
    (context->coreComponents[component].level >= level) )
 
+
+#define zowelog(context, compID, level, formatString, ...) \
+        _zowelog(context, compID, __FILE__, __LINE__, level, formatString, ##__VA_ARGS__);
```

Comment:
By introducing a macro we will break existing plugins because they assume that zowelog has fewer args. It may be better to have `zowelog2` to keep the backward compatibility.